src/mixins/payment-mixin.ts

```diff
@@ -71,8 +71,9 @@ export class PaymentMixin extends Mixins(ActionMixin) {
     return this.$store.getters[paymentTypes.GET_PAYMENT_RECEIPT]
   }
 
-  get paymentFees () {
-    return this.$store.getters[paymentTypes.GET_PAYMENT_FEES]
+  get paymentFees (): any[] {
+    const fees = this.$store.getters[paymentTypes.GET_PAYMENT_FEES]
+    return [fees]
```

Comment:
This is the fix for "paymentFees is not iterable".

The root cause is that several Vue components pass `paymentFees` to FeeSummary.vue, which expects an array of fee objects. However, `fees` is [stored as an object, not an array](https://github.com/bcgov/namerequest/blob/c8e8b6722adca77bccaca97d18c934e8506a2045/src/modules/payment/store/index.ts#L36). So, this code change now provides "paymentFees" as an array of a single object, and everyone is happy.

Reply:
I'm guessing that the previous version of Typescript handled an object as equivalent to an array of a single object, which is why this worked before.

Here it is working now:

![image](https://github.com/bcgov/namerequest/assets/10002889/bf111eba-ad4b-4ec6-96d9-352a195d2128)